src/contracts/core/AVSDirectory.sol

```diff
@@ -127,20 +355,59 @@ contract AVSDirectory is
     }
 
     /**
-     * @notice Called by an avs to emit an `AVSMetadataURIUpdated` event indicating the information has updated.
-     * @param metadataURI The URI for metadata associated with an avs
+     *
+     *                         INTERNAL FUNCTIONS
+     *
      */
-    function updateAVSMetadataURI(string calldata metadataURI) external {
-        emit AVSMetadataURIUpdated(msg.sender, metadataURI);
+
+    /**
+     * @notice Helper function used by migration & registration functions to register an operator to operator sets.
+     * @param avs The AVS that the operator is registering to.
+     * @param operator The operator to register.
+     * @param operatorSetIds The IDs of the operator sets.
+     */
+    function _registerToOperatorSets(address avs, address operator, uint32[] calldata operatorSetIds) internal {
```

Comment:
Nit: could we order params as operator, avs, opSetIds? More consistent with interfaces and I prefer having AVS and operatorSetIds next to eachother as the operatorSet.